Tests/MusicXMLTests/LilyPondTests/ArticulationsTextsTests.swift

```diff
@@ -25,13 +25,13 @@ class ArticulationsTextsTests: XCTestCase {
         let expected = Direction(
             [
                 .words(
-                    FormattedText("Normal, Small, Colored, Below",
-                        printStyle: PrintStyle(
-                            position: Position(defaultX: 1, defaultY: -15),
-                            font: Font(size: FontSize(.small)),
-                            color: Color(hexString: "#FF8000")
+                    [FormattedText("Normal, Small, Colored, Below",
```

Comment:
Indentation was weird here, but I think it's still a little weird. How about something like this: 

```Swift
[
    FormattedText("Normal, Small, Colored, Below",
        printStyle: PrintStyle(
            position: Position(defaultX: 1, defaultY: -15),
            font: Font(size: FontSize(.small)),
            color: Color(hexString: "#FF8000")
        )
    )
]
```